vendor/github.com/onsi/ginkgo/ginkgo/watch/delta.go

```diff
@@ -14,7 +14,7 @@ type DescendingByDelta []*Suite
 
 func (a DescendingByDelta) Len() int           { return len(a) }
 func (a DescendingByDelta) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
-func (a DescendingByDelta) Less(i, j int) bool { return a[i].Delta() > a[j].Delta() }
+func (a DescendingByDelta) Less(i, j int) bool { return a[i].Delta() < a[j].Delta() }
```

Comment:
Please follow https://github.com/kubernetes/community/blob/master/contributors/devel/godep.md to update vendor dependencies.